notebooks/Simulated_Data_MediumFunction_Tutorial.html

```diff
@@ -13296,7 +13296,7 @@ <h2 id="User-parameters">User parameters<a class="anchor-link" href="#User-param
 <div class="prompt input_prompt">In&nbsp;[9]:</div>
 <div class="inner_cell">
     <div class="input_area">
-<div class=" highlight hl-ipython3"><pre><span></span><span class="n">user_params</span><span class="p">[</span><span class="s1">&#39;output_directory&#39;</span><span class="p">]</span> <span class="o">=</span> <span class="s1">&#39;../results/&#39;</span>
+<div class=" highlight hl-ipython3"><pre><span></span><span class="n">user_params</span><span class="p">[</span><span class="s1">&#39;output_dir&#39;</span><span class="p">]</span> <span class="o">=</span> <span class="s1">&#39;../results/&#39;</span>
```

Comment:
Did you regenerate the html versions of the notebooks? Seem like only the output_dir changes were stored.

Reply:
Good catch!  Possible this file was updated as a side-effect of opening the file in PyCharm.  Assuming the base notebook changes are ok, it's worth re-running them to update the HTML.

Reply:
Force pushing to update the HTML files after re-running, and to copy the latest `ART_tutorial.ipynb` from the private repo (PR#249).  No changes to the already-reviewed notebooks.